<ui version="4.0" >
 <class>VolumeEdit</class>
 <widget class="QWidget" name="VolumeEdit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayout" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>40</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="NameLabel" >
      <property name="text" >
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="mNameEdit" />
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>260</y>
     <width>381</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="mOKButton" >
      <property name="text" >
       <string>&amp;OK</string>
      </property>
      <property name="default" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mApplyButton" >
      <property name="text" >
       <string>&amp;Apply</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mCancelButton" >
      <property name="text" >
       <string>&amp;Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayout_2" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>70</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Path:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="mPathEdit" />
    </item>
    <item>
     <widget class="QPushButton" name="pushButton" />
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
